modules/rostests/apitests/gdi32/GetTextMetrics.c

```diff
@@ -464,6 +517,8 @@ START_TEST(GetTextMetrics)
         {
             const TEST_ENTRY *test = &font->tests[k];
 
+            printf("Testing '%s'.\n", font->entry_name);
```

Comment:
please use `trace` here, which is a builtin winetest function (and can be configured with test parameters)

Reply:
Done and moved to a better location as well. Thanks.